specs/awscli-2.spec

```diff
@@ -29,42 +29,37 @@ Obsoletes:      python3-botocore <= 1
 This package provides version 2 of the unified command line
 interface to Amazon Web Services.
 
+
 %prep
-%autosetup -n %{srcname}-%{version}
+%autosetup -p1 -n %{srcname}-%{version}
 
 %if %{with examples}
 find awscli/examples/ -type f -name '*.rst' -executable -exec chmod -x '{}' +
 %else
-rm -vr awscli/examples
+rm -r awscli/examples
 %endif
 
+%generate_buildrequires
+%pyproject_buildrequires -r
+
 %build
-# cd %{srcname}-%{version}
-%py3_build
+%pyproject_wheel
+
 
 %install
-%py3_install
-rm -vf %{buildroot}%{_bindir}/{aws_bash_completer,aws_zsh_completer.sh,aws.cmd}
-install -Dpm0644 bin/aws_bash_completer \
-  %{buildroot}%{_datadir}/bash-completion/completions/aws
-install -Dpm0644 bin/aws_zsh_completer.sh \
-  %{buildroot}%{_datadir}/zsh/site-functions/_awscli
-install -Dpm0644 doc/* %{buildroot}%{_datadir}/
-
-%files
-%doc %{name}/doc/README.rst
-%license %{name}/LICENSE.txt
+%pyproject_install
+
+%pyproject_save_files awscli
+
+%files -f %{pyproject_files}
+%doc README.rst
+
 %{_bindir}/aws
+%{_bindir}/aws.cmd
+%{_bindir}/aws_bash_completer
 %{_bindir}/aws_completer
-%{_bindir}/aws_legacy_completer
-%{python3_sitelib}/awscli/
-%{python3_sitelib}/awscli-%{version}-*.egg-info/
-%dir %{_datadir}/bash-completion
-%dir %{_datadir}/bash-completion/completions
-%{_datadir}/bash-completion/completions/aws
-%dir %{_datadir}/zsh
-%dir %{_datadir}/zsh/site-functions
-%{_datadir}/zsh/site-functions/_awscli
+%{_bindir}/aws_zsh_completer.sh
+
 
 %changelog
 * Fri Mar 13 2020 David Duncan <davdunc@amazon.com> - 2.0.3-2
```

Comment:
Need to bump the spec on the changelog to add your name in here.

Reply:
I'll send up a follow up PR to do that